src/test/scala/org/kibanaLoadTest/helpers/Helper.scala

```diff
@@ -273,8 +273,16 @@ object Helper {
     }
 
     val metaJson = Helper.getMetaJson(testRunFilePath, simLogFilePath)
+    val usersStatsJsonString: String = s"""
+      {
+      "usersCountTotal": ${usersStats.count},
+      "avgUserSessionTime": ${usersStats.avgSessionTime}
```

Comment:
@marius-dr I guess ES will handle it on its own, I will check types mapping after first ingestion

Reply:
we can update the mapping before the PR gets merged so we make sure it's what type you want them to be: double, int?

Reply:
Int is too small for `avgUserSessionTime`, but they both will be rounded

Reply:
wrong index @marius-dr :)